Composer/packages/server/src/controllers/project.ts

```diff
@@ -347,6 +347,51 @@ async function removeFile(req: Request, res: Response) {
   }
 }
 
+async function updateManifestFile(req: Request, res: Response) {
+  const projectId = req.params.projectId;
+  const user = await ExtensionContext.getUserFromRequest(req);
+  const currentProject = await BotProjectService.getProjectById(projectId, user);
+  if (currentProject !== undefined) {
+    const lastModified = await currentProject.updateManifestLuFile(req.body.name, req.body.content);
+    res.status(200).json({ lastModified: lastModified });
+  } else {
+    res.status(404).json({
+      message: 'No such bot project opened',
```

Comment:
404 means not found, error message should match that

Reply:
Use `      message: 'No such bot project found',`